src/code/z_scene.c

```diff
@@ -403,17 +383,14 @@ void func_80099090(GlobalContext* globalCtx, SceneCmd* cmd) {
     globalCtx->envCtx.windSpeed = cmd->windSettings.unk_07;
 }
 
-// Scene Command 0x13: Exit List
-void func_800990F0(GlobalContext* globalCtx, SceneCmd* cmd) {
+void Scene_CommandExitList(GlobalContext* globalCtx, SceneCmd* cmd) {
     globalCtx->setupExitList = SEGMENTED_TO_VIRTUAL(cmd->exitList.segment);
 }
 
-// Scene Command 0x09: Undefined
-void func_80099134(GlobalContext* globalCtx, SceneCmd* cmd) {
+void Scene_CommandUndefined9(GlobalContext* globalCtx, SceneCmd* cmd) {
 }
 
-// Scene Command 0x15: Sound Settings
-void func_80099140(GlobalContext* globalCtx, SceneCmd* cmd) {
+void Scene_CommandSoundSettings(GlobalContext* globalCtx, SceneCmd* cmd) {
     globalCtx->sequenceCtx.seqId = cmd->soundSettings.seqId;
     globalCtx->sequenceCtx.natureAmbienceId = cmd->soundSettings.natureAmbienceId;
```

Comment:
Just making a note:
`Scene_CommandSoundSettings` -> `Scene_CommandSequenceSettings`
`soundSettings` -> `sequenceSettings`
I can always do this later though, there's a few `sound` -> `sequence` changes around here to make